src/js/wall/LoginWall.tsx

```diff
@@ -14,7 +14,9 @@ export function LoginWall({ reset }: LoginWallProps) {
     return (
         <WallContainer>
             <WallDialog>
-                <WallLoginContainer>{reset ? <ResetForm /> : <LoginForm />}</WallLoginContainer>
+                <WallLoginContainer>
+                    {reset ? <ResetForm /> : <LoginForm error="Invalid username or password" />}
```

Comment:
```suggestion
                    {reset ? <ResetForm /> : <LoginForm  />}
```

Generally speaking errors will either be defined by the component itself (ex. form validation) or sent from the API. 

Since this is an API error the message should be derived from the API whenever possible. In cases where it is not possible the error should be specified at the lowest level of the component tree that makes sense. 

In this case I think you can simply get the error message from the API response.